os/se/sss/Kconfig

```diff
@@ -49,4 +49,16 @@ config SE_STORAGE
 	default n
 	---help---
 		SE supports Secure Stroage
-		
\ No newline at end of file
+
+config DEBUG_SECURITY_ISP_WRAPPER_ERROR
+	bool "SECURITY HAL ISP WRAPPER ERROR DEBUG"
+	default n
+	---help---
+		Enable SECURITY ISP WRAPPER ERROR DEBUG
+
+config DEBUG_SECURITY_ISP_WRAPPER_INFO
+	bool "SECURITY HAL ISP WRAPPER INFO DEBUG"
+	default n
+	---help---
```

Comment:
Need to make a dependency with DEBUG_VERBOSE

Reply:
Applied. Thanks for the comment!